src/store/state/state-model.ts

```diff
@@ -27,6 +27,8 @@ export const stateModel: StateModelIF = {
     identifier: ''
   },
   nameRequest: {
+    legalType: '',
```

Comment:
Do you have a type for "none"? (or perhaps null?)

Reply:
i think null makes sense here, will update!

Reply:
Actually, i think it is ok to initialize as an empty string here isn't it? The base type is a string

Reply:
This is fine.
I can't remember what I was thinking now.
Carry on.